.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-npx lint-staged
+npx lint-staged --no-stash
```

Comment:
according to their docs, lint-staged uses the stash to IMPROVE speed. what are we seeing here that differs from that?

Reply:
https://github.com/okonet/lint-staged/issues/715

Reply:
cool, works for me